scenarios/random-selection-scenario.js

```diff
@@ -13,6 +13,7 @@ function getEnigmaConfig(sessionId) {
 
   const headers = config.headers || {};
   headers['X-Qlik-Session'] = sessionId;
+  headers['User-Agent'] = 'Mozilla/5.0+(Macintosh;+Intel+Mac+OS+X+10_14_3)+AppleWebKit/537.36+(KHTML,+like+Gecko)+Chrome/71.0.3578.98+Safari/537.36';
```

Comment:
Don't think this should be hardcoded. Can be added in the config if it's needed.